index.rst

```diff
@@ -372,30 +372,41 @@ c.l.o stubb
 
   Here is where we currently are in the release process. Current step in bold.
 
-  Precursor Steps
+  Release Precursor Steps
   ---------------------------------
 
   1. Identify any pre-release blockers ("must-have features") :tools:
```

Comment:
What is `:tools:` supposed to signify?

Reply:
this shows an icon representing tools. I don't use it during the process... but I suppose it was used in the past to indicate in which point we are working right now.